src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,38 @@
 
 class Animal {
   // write your code here
+  static alive = [];
```

Comment:
The `alive` array should be a private static property according to the task requirements. Consider using a private field to encapsulate it.